src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -85,67 +79,82 @@ const ResearchProfile = () => {
   ) : !person ? (
     <NotFound />
   ) : (
-    <BackgroundDiv>
-      <Helmet>
-        <title>{fullName}</title>
-      </Helmet>
-      <Typography variant="h1" sx={{ mt: '1rem', mb: '2rem' }}>
-        {fullName}
-      </Typography>
-      <Typography variant="h2">{t('common.employments')}</Typography>
-      {activeAffiliations.length > 0 ? (
-        <Box sx={lineSx}>
-          <WorkIcon />
-          <Box sx={textContainerSx}>
-            {activeAffiliations.map(({ organization }) => (
-              <AffiliationHierarchy key={organization} unitUri={organization} commaSeparated />
+    <Box>
```

Comment:
Antar dette bare er for å samle ting layoutmessig. I så fall kan vi bruke en div-tag i stedet for en MUI-komponent . selv om det i praksis blir det samme uansett, men ligger litt ekstragreier i Box vi ikke trenger her da
```suggestion
    <div>
```